src/NBitcoin/Network.cs

```diff
@@ -378,7 +474,7 @@ public static IBitcoinString Parse(string str, Network expectedNetwork)
                             candidate = new BitcoinWitScriptAddress(str, network);
 
                         if (candidate is T)
-                            return (T) (object) candidate;
+                            return (T)(object)candidate;
```

Comment:
casting to object is redundant

Reply:
why did you pick this one lol.
ok, fixed

Reply:
>why did you pick this one lol.

Well, had to find at least something what is wrong with the PR 😂